smact/utils/crystal_space/generate_composition_with_smact.py

```
@@ -13,11 +13,30 @@
 from smact import Element, ordered_elements
+from smact.data_loader import lookup_element_oxidation_states_custom
 from smact.screening import smact_filter
 
 warnings.simplefilter(action="ignore", category=UserWarning)
 
 
+oxidation_states_set_global = None
+max_stoich_global = None
+
+
+def init_worker(ox_states, max_stoich):
```

Ruff (0.11.9)

Missing docstring in public function

(D103)